librz/core/rop.c

```diff
@@ -565,11 +579,28 @@ static int fill_rop_gadget_info_from_events(RzCore *core, RzRopGadgetInfo *gadge
 			break;
 		}
 		if (is_dependency && curr_event) {
+			bool is_stack_evt = false;
 			// shouldn't take this path if it is a dependency
```

Comment:
This comment doesn't make sense with the if statement above. It shouldn't be a dependency but it checks if it is a dependency?

Reply:
Removed it :)